### ISSUE TITLE
stop when parsing ipv6 adresses.

### ISSUE THREAD

**Issue by Cyrix126** [Rucknium/xmrpeers#1]
(bitmonero dir is modified).

monerod options:

```
./monerod --zmq-pub tcp://0.0.0.0:18083 --out-peers 32 --in-peers 64 --add-priority-node=p2pmd.xmrvsbeast.com:18080 --disable-dns-checkpoints --enable-dns-blocklist --data-dir /media/hdd/monero_node --rpc-use-ipv6 --p2p-use-ipv6 --rpc-bind-ip 0.0.0.0  --confirm-external-bind --log-level=net.p2P.msg:INFO
```

The program stop with the following error after some times.

```
Erreur dans pingr::ping_port(ip, port = port, count = ping.count) :                                                
  Cannot resolve host name
De plus : Messages d'avis :
Dans pingr::ping_port(ip, port = port, count = ping.count) :
  NAs introduits lors de la conversion automatique
```

Maybe is it because of ipv6 addresses ? looking at the code it seems only ipv4 is expected.

**Comment by Rucknium** [comment]
Thank you for the bug report! It is probably fixed with b4bd6089e7ef68d402a43bccbd249dbef80c20d8

To update to the latest version, in an R terminal:
```R
remotes::install_github("Rucknium/xmrpeers")
```
It may ask if you want to update packages from CRAN. In this situation you can choose to not update unless you like to watch software compile. Then quit R with
```R
quit("no")
```
Start R again with `R` in the terminal. Then start the pinging script again:
```R
xmrpeers::ping.peers(bitmonero.dir = "<log_directory>")
```

**Comment by Cyrix126** [comment]
seems to be working fine.
